sdk/storage/storage-blob/samples/typescript/advanced.ts

```diff
@@ -61,7 +61,13 @@ async function main() {
   // Create a container
   const containerName = `newcontainer${new Date().getTime()}`;
   const containerClient = blobServiceClient.getContainerClient(containerName);
-  await containerClient.create();
+  try {
+    await containerClient.create();
+  } catch (err) {
+    console.log(
+      `Creating a container fails, requestId - ${err.details.requestId}, statusCode - ${err.statusCode}, errorCode - ${err.details.errorCode}`
```

Comment:
Just catching the errors and console.logging so that the rest of the sample is executed :)

Reply:
We are creating a new container using the current time in its name so unlikely it would fail with container-already-exists error? Other errors might not be okay to ignore.